### DISCUSSION TITLE
Is it possible to see the list of all jobs waiting for a runner across an entire organization

### DISCUSSION THREAD

**Discussion by Fgerthoffert** [community/community#160491] [Actions]
Hi,

We are trying to debug and understand why some of our jobs are left without runners and get stuck on the following message:

```
Requested labels: self-hosted
Job defined at: [REDACTED]/.github/workflows/weekly-manual-run.yml@refs/heads/main
Waiting for a runner to pick up this job...
```

While I have a question opened in [terraform-aws-github-runner](https://github.com/philips-labs/terraform-aws-github-runner/discussions/2931) since the issue/misconfiguration might be on that end, I was wondering if there is an API at GitHub that would allow me to fetch more details about the queue of jobs at an org level ?

I basically would like to find a way to figure out the jobs that have been waiting for a runner for more than 5mn across our entire org.

Thanks

**Comment by taghoutikeyrus** [comment]
Any solution?

**Comment by 4c74356b41** [comment]
any updates here?

**Comment by guilhermepim-taxa** [comment]
I am not sure that exists, unfortunately.. from past experiences, what I think you'd have to do is poll each specific repository and their workflow runs, and then their jobs to see which has not started

An alternative might be listening for webhooks on workflow_jobs and storing them somewhere.. then you can see if there have been jobs which have been queued but not picked up.. 

not the ideal scenario I understand, but I think that's how it is right now 😰

**Comment by seanlafferty-ibm** [comment]
I've been looking for this capability as a means to control the scaling of my self-hosted runners. My runners are not hosted publicly so I cannot respond to github webhook events as suggested in https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/autoscaling-with-self-hosted-runners

If I could get the queued jobs org wide via polling, I could scale my locally-hosted runners appropriately.
